frontend/src/components/placementVisuals/AddPlacementContainer.tsx

```diff
@@ -58,12 +59,29 @@ const AddPlacementContainer = ({ onConfirm, existingContainers }: AddPlacementCo
         if (container.samples.length > 0) {
             const loadedSamples: FMSSample[] = (await dispatch(api.samples.list({ id__in: container.samples.join(','), limit: 100000 }))).data.results
 
+            const projectIDs = new Set(loadedSamples.map(sample => sample.project))
+            const promises: Promise<any>[] = []
+            for (const projectID of projectIDs) {
+                if (projectID) {
+                    promises.push(dispatch(getProject(projectID)))
+                }
+            }
+            await Promise.all(promises)
+
+            const coordinates = selectCoordinatesByID(store.getState())
+            const projects = selectProjectsByID(store.getState())
+
             loadedSamples.forEach(sample => {
-                newDestination[sample.id] = { id: sample.id, coordinates: (sample.coordinate && coordinates[sample.coordinate].name) as string, name: sample.name }
+                newDestination[sample.id] = {
+                    id: sample.id,
+                    coordinates: sample.coordinate ? coordinates[sample.coordinate]?.name ?? '...' : 'undefined',
+                    name: sample.name,
+                    project: sample.project ? projects[sample.project]?.name ?? '...' : 'undefined'
```

Comment:
... ?! the string 'undefined'  ... what about leaving it empty ? and not 'empty' ... or at least ''.

Reply:
Fair enough. I just want the user to know that there is **definitely** no project associated with the sample.

Reply:
ok well it's not that important. we should have some standard way to say there is none ... i think some places we use "-". Not sure what would be best ...The value of the project name can be Any typical character combination so even "NA" and "undefined" are valid project names ... maybe a special icon we could use for that ?